docs/platform/block-storage/block-storage-use-cases/index.md

```diff
@@ -8,58 +8,80 @@ license: '[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0)'
 published: 2019-08-27
 modified_by:
   name: Linode
-title: "Block Storage Use Cases"
+title: "Use Cases for Block Storage"
 contributor:
   name: Linode
 ---
 
 ## What is Block Storage
 
-Block storage is a method of storing data. With Block Storage, data is divided and stored in small *blocks*. The Block Storage *Volume* that houses these blocks is mounted to a file system and acts like a hard drive, providing scalable amounts of additional raw storage capacity to the server it's attached to. Because the Volume is directly connected to the server, the I/O speeds are much faster than those of an alternative storage solution like Object Storage. Additionally, all data stored with Linode Block Storage is replicated three times, creating data that is highly available and fault tolerant. All that's required to get started using Block Storage is to mount the Volume and format it with the file system of your choice.
+Block Storage is a type of persistent cloud data storage that is similar to a traditional block device, like the hard drive in a PC. With Block Storage, your data is divided into *blocks*, which are the small, discrete units that Block Storage can read from and write to. These blocks are assigned unique identifiers, but these are generally not human-readable, so a filesystem is usually installed which maps your files to the underlying blocks they correspond to. This relationship is also analogous to your PC's filesystem and hard drive.
```

Comment:
I expanded on this a good deal and added a benefits & limitations section. If we think this is too much, we can definitely pare it back. I found a home for all the content in the original paragraph in the various parts of these two sections